src/ctim/schemas/coa.clj

```diff
@@ -39,8 +69,8 @@
                          " one or more related courses of action"))
      ;; Not provided: handling
      ;; Not provided: parameter_observables ;; Technical params using the CybOX language
-     ;; Not provided: structured_COA ;; actionable structured representation for automation
-     })))
+     :structured_coa OpenC2COA})))
```

Comment:
So, like the conditional problem below, we need to handle different types of structured COAs.

StructuredCOA should have a type value, and then a key based on that type.  So `{:structured_coa_type "openc2" :open_c2_coa {...}}`

This way it can have multiple types of structures/specification.

Also, on the whole FOO_type vs "type" I am partial to the :foo_type - mostly because in ES, we could not have different mappings for :type fields in different locations in the object, they would all have to be string/enum.  That's my only argument.  I do think it introduces some verbosity tho, and makes writing a generic structured accessor harder.